<launch>
    <node pkg="scqn_sam" type="scqn_sam_node" name="scqn_sam_node" output="screen">
        <rosparam file="$(find scqn_sam)/launch/scqn_sam_params.yaml" command="load" />
    </node>

	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find scqn_sam)/r.rviz" />

</launch>